Packs/Core/ReleaseNotes/3_0_52.md

```diff
@@ -0,0 +1,15 @@
+
+<!-- 
+#### Integrations
+
+#### Integrations
+
+##### Indicators detection
+
+- No changes related directly to this integration.
+
+##### Investigation & Response
+
+- No changes related directly to this integration.
```

Comment:
```suggestion
No changes related directly to this integration.

##### Investigation & Response

No changes related directly to this integration.
```
If there are no changes, why do we need to list that?

Reply:
since changes were made to "Investigation & Response" dependent pack (CoreIRApiModule ) a new version needs to be released.